new-site/src/docs/elements/components/buttons/index.mdx

```diff
@@ -20,26 +17,18 @@ import Usage from './usage.mdx';
 </StatusLabel>
 
 <LeadParagraph>
-Buttons are meant to make actions easily visible and understandable to the user. HDS offers different kinds of button variations which each suit for different needs.
+  Buttons are meant to make actions easily visible and understandable to the user. HDS offers different kinds of button
+  variations which each suit for different needs.
 </LeadParagraph>
 
-<Tabs>
-  <Tabs.TabList className="pageTabsList">
-    <Tabs.Tab>Usage</Tabs.Tab>
-    <Tabs.Tab>Code</Tabs.Tab>
-    <Tabs.Tab>Accessibility</Tabs.Tab>
-    <Tabs.Tab>Customisation</Tabs.Tab>
-  </Tabs.TabList>
-  <Tabs.TabPanel>
-    <Usage />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Code />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Accessibility />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Customisation />
-  </Tabs.TabPanel>
-</Tabs>
+<PageTabs pageContext={props.pageContext}>
```

Comment:
This looks fine to me and it's not too hard to transform all Tabs into this format.

Just wondering, did we investigate building this support directly on the Tabs component? Would it be possible?

Reply:
I think it might not be possible since it needs really custom children mapping.